CefSharp.Wpf/HwndHost/Internals/DelegateCommand.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright © 2019 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+using System;
+using System.Windows.Input;
+
+namespace CefSharp.Wpf.HwndHost.Internals
+{
+    /// <summary>
+    /// DelegateCommand
+    /// </summary>
+    /// <seealso cref="System.Windows.Input.ICommand" />
+    internal class DelegateCommand : ICommand
```

Comment:
Can this be removed in favor of
https://github.com/cefsharp/CefSharp/blob/a81202b5a0fa6c83bed0850230357ebfd093189f/CefSharp.Wpf/DelegateCommand.cs#L14 ?

Reply:
Excellent suggestion! Thanks.